wp_rs_cli/src/bin/wp_rs_cli/main.rs

```diff
@@ -182,7 +182,7 @@ impl SimplifiedDiscoveryResult {
                 println!("{}", format!("Login URL found: {login_url}").green());
             }
             Err(error) => {
-                println!("{}", format!("Error: {}", error).red());
+                println!("{}", error.to_string().red());
```

Comment:
This is just a little thing – it was weird having the string read `Error: Unable to connect to server: ⁨Connection reset by peer`

It's printed in red anyway, so I think folks will get that it's an error